src/main/resources/l10n/general_en.properties

```diff
@@ -7,10 +7,13 @@ msg.group.display=Display Name
 
 msg.user.create=Create User
 msg.user.dialog.deactivate.message=Are you sure, that you want to deactivate the user?
+msg.user.dialog.deactivate.selected.message=Are you sure, that you want to deactivate the selected users?
```

Comment:
I think you have to leave off the comme here. also you can remove the 'that', but that's optional. This is also true for the following lines.